.github/workflows/reusable_build_docker.yaml

```diff
@@ -0,0 +1,131 @@
+# This is a reusable workflow used by master and release CI
+on:
+  workflow_call:
+    inputs:
+      arch:
+        description: x86_64 or aarch64
+        required: true
+        type: string
+      tagname:
+        description: master or tag name
+        required: true
+        type: string
+      bucket:
+        description: bucket suffix for packages
+        required: false
+        default: ''
+        type: string
+      version:
+        description: 'Falco version extracted from userspace/falco/config_falco.h'
+        required: true
+        type: string
+
+jobs:
+  build-docker:
+    # See https://github.com/actions/runner/issues/409#issuecomment-1158849936
+    runs-on: ${{ (inputs.arch == "aarch64") && fromJSON('[ "self-hosted", "linux", "ARM64" ]') || 'ubuntu-latest' }}
```

Comment:
Here we are trying to run a container on our runner that then needs to use docker to build and push images. I don't think this is feasible :/ 
We need to find another way to do this (perhaps splitting the jobs for x86 and arm and going natively on our node?)